com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Raytracing/HDRaytracingManager.HDRTASManager.cs

```diff
@@ -0,0 +1,270 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine.Experimental.Rendering;
+using UnityEngine.Experimental.Rendering.RenderGraphModule;
+
+namespace UnityEngine.Rendering.HighDefinition
+{
+    struct HDEffectsParameters
+    {
+        public bool shadows;
+        public bool ambientOcclusion;
+        public int aoLayerMask;
+        public bool reflections;
+        public int reflLayerMask;
+        public bool globalIllumination;
+        public int giLayerMask;
+        public bool recursiveRendering;
+        public int recursiveLayerMask;
+        public bool subSurface;
+        public bool pathTracing;
+        public int ptLayerMask;
+
+        // Flag that tracks if at least one effect is enabled
+        public bool rayTracingRequired;
+    };
+
+    class HDRTASManager
+    {
+        public RayTracingAccelerationStructure rtas = null;
+        public RayTracingInstanceCullingConfig cullingConfig = new RayTracingInstanceCullingConfig();
+        public List<RayTracingInstanceCullingTest> instanceTestArray = new List<RayTracingInstanceCullingTest>();
+
+        // Culling tests
+        RayTracingInstanceCullingTest ShT_CT = new RayTracingInstanceCullingTest();
```

Comment:
Sorry but from the comment I don't understand why this code is here? 
But I guess "Test" here doesn't mean "Testing" right? i.e not part of our test framework

Reply:
yeah not part of the test framework, it's just the set of tests that are used to inject into the RTAS @sebastienlagarde